packages/ui/src/theme/tokens/components/stepperField.ts

```diff
@@ -1,21 +1,65 @@
 import {
+  ColorValue,
   DesignToken,
+  FontSizeValue,
   FlexDirectionValue,
   TextAlignValue,
+  BorderColorValue,
 } from '../types/designToken';
 
 interface StepperFieldInputTokens {
   textAlign: DesignToken<TextAlignValue>;
+  color: DesignToken<ColorValue>;
+  fontSize: DesignToken<FontSizeValue>;
+}
+
+interface ButtonStateColorTokens {
+  color: DesignToken<ColorValue>;
+  backgroundColor: DesignToken<ColorValue>;
 }
 
 export interface StepperFieldTokens {
+  borderColor: DesignToken<BorderColorValue>;
   flexDirection: DesignToken<FlexDirectionValue>;
   input: StepperFieldInputTokens;
+  button: {
+    color: DesignToken<ColorValue>;
+    backgroundColor: DesignToken<ColorValue>;
+    _active: ButtonStateColorTokens;
+    _focus: ButtonStateColorTokens;
+    _disabled: ButtonStateColorTokens;
+    _hover: ButtonStateColorTokens;
+  };
 }
 
 export const stepperfield: StepperFieldTokens = {
+  borderColor: { value: '{components.fieldcontrol.borderColor}' },
   flexDirection: { value: 'column' },
   input: {
+    color: { value: '{components.fieldcontrol.color}' },
+    fontSize: { value: '{components.fieldcontrol.fontSize}' },
     textAlign: { value: 'center' },
   },
+  button: {
+    color: { value: '{components.button.color}' },
+    backgroundColor: { value: 'transparent' },
```

Comment:
Looks like we only have `color.transparent`. How about adding a `color.background.transparent`?

Reply:
thanks this has been updated to the token